terraform/modules/cos/README.md

```diff
@@ -44,7 +62,83 @@ Users should ensure that Terraform is aware of the `juju_model` dependency of th
 
 To deploy this module with its needed dependency, you can run `terraform apply -var="model_name=<MODEL_NAME>" -auto-approve`. This would deploy all COS HA solution modules in the same model.
 
-### High Availability 
-
-By default, this Terraform module will deploy each worker with `1` unit. To configure the module to run `x` units of any worker role, you can run `terraform apply -var="model_name=<MODEL_NAME>" -var="<ROLE>_units=<x>" -auto-approve`.
-See each ... for the recommended scale for each role.
+### High Availability
+
+By default, this Terraform module will deploy each worker with `1` unit. If you want to scale each Loki, Mimir or Tempo worker unit please check the variables available for that purpose in `variables.tf`. For instance to deploy 3 units of each Loki worker, you can run:
+
+```shell
+terraform apply -var='minio_password=Password' -var='minio_user=User' -var='model_name=test'\
+-var='loki_backend_units=3' -var='loki_read_units=3' -var='loki_write_units=3'
+```
+
+
+### Sample deployment
+
+```hcl
+terraform {
+  required_version = ">= 1.5"
+  required_providers {
+    juju = {
+      source  = "juju/juju"
+    }
+  }
+}
+
+module "cos" {
+  source     = "git::https://github.com/canonical/observability//terraform/modules/cos"
+  model_name = var.model_name
+}
+
+# Assumes that model already exists
+variable "model_name" {
+  type    = string
+}
+
+
+resource "juju_application" "minio" {
```

Comment:
Just making sure: in this PR there is a new `s3` module, but here we're not using it.
So just confirming again that this snipped is indeed up to date.